.github/workflows/build-python-wheels.yml

```diff
@@ -56,6 +59,7 @@ jobs:
 
       - uses: eiennohito/gha-manylinux-build@master
```

Comment:
[`eiennohito/gha-manylinux-build`](https://github.com/eiennohito/gha-manylinux-build) does not have `arch` input.
I guess we need a different method to build aarch64 wheel.

Reply:
I can update the build definition if you will make a PR for it. Or you can clone the repo to your user and use the clone definiton.